tracer/test/test-applications/integrations/Samples.MSTestTests2/TestSuite.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections;
+using System.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Samples.MSTestTests2
```

Comment:
This is the same code as MsTestTests, right? If so, then we shouldn't duplicate it, just link it in to the project?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/5269/commits/c400bda63c9512ec05e6a0a2551dbe6c8e92c123

Reply:
Haha, I broke you 😆